www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/index.tsx

```diff
@@ -208,7 +215,10 @@ const ExportablePerformanceGraphWithTimeline = ({
         <PerformanceGraph
           endpoint={getEndpoint()}
           graphHeight={graphHeight}
-          xAxisTickFormat={selectedTimePeriod.dateTimeFormat}
+          xAxisTickFormat={
+            selectedTimePeriod?.dateTimeFormat ||
+            customTimePeriod.xAxisTickFormat
```

Comment:
```suggestion
          axisTickFormat={
            selectedTimePeriod?.dateTimeFormat ||
            customTimePeriod.axisTickFormat
```

Reply:
The "x" is to specify to "X" axis. Because "axisTickFormat" means overall axis (X and Y)